clang/test/Driver/sycl-spirv-ext.c

```diff
@@ -0,0 +1,108 @@
+// REQUIRES: system-linux
+// -------
+// Generate .o file as SYCL device library file.
+//
+// RUN: touch %t.devicelib.cpp
+// RUN: %clang %t.devicelib.cpp -fsycl -fsycl-targets=spir64-unknown-unknown -c --offload-new-driver -o %t_1.devicelib.o
+// RUN: %clang %t.devicelib.cpp -fsycl -fsycl-targets=spir64_gen-unknown-unknown -c --offload-new-driver -o %t_2.devicelib.o
+// RUN: %clang %t.devicelib.cpp -fsycl -fsycl-targets=spir64_x86_64-unknown-unknown -c --offload-new-driver -o %t_3.devicelib.o
+
+/// Check llvm-spirv extensions that are set
+
+// RUN: %clang -target x86_64-unknown-linux-gnu -fsycl --offload-new-driver \ 
+// RUN:   -fsycl-targets=spir64-unknown-unknown -c %s -o %t_1.o
+// RUN: clang-linker-wrapper -sycl-device-libraries=%t_1.devicelib.o \
+// RUN:   "--host-triple=x86_64-unknown-linux-gnu" "--linker-path=/usr/bin/ld" \
+// RUN:   "--" HOST_LINKER_FLAGS "-dynamic-linker" HOST_DYN_LIB "-o" "a.out" \
+// RUN:   HOST_LIB_PATH HOST_STAT_LIB %t_1.o --dry-run 2>&1 \
```

Comment:
For my education - how are `HOST_LINKER_FLAGS`, `HOST_LIB_PATH` and `HOST_STAT_LIB` populated?

Reply:
Good question. They are placeholders for some random flags and paths to check if we are passing the right set of options to host linker, but they do not have any reason to exist in this test and can be safely removed.

Thanks